mewarpx/mewarpx/mcc_wrapper.py

```diff
@@ -0,0 +1,90 @@
+import glob, os
+from pywarpx import picmi
+from mewarpx import util as mwxutil
+
+# For use later to sync with diode test template and access sim object in mwxrun
+from mewarpx import mwxrun
+
+
+class MCC():
+    """Wrapper used to initialize MCC parameters"""
+
+    def __init__(self, electron_species, ion_species, P_INERT,
+                 T_INERT, scraper=None, **kwargs):
+        """Initialize MCC"""
+        self.electron_species = electron_species
+        self.ion_species = ion_species
+        self.P_INERT = P_INERT
+        self.T_INERT = T_INERT
+        self.N_INERT = mwxutil.ideal_gas_density(self.P_INERT, self.T_INERT)
+        self.scraper = scraper
+
+        # include all collision processes that match species
+        path_name = os.path.join("../../../warpx-data/MCC_cross_sections", self.ion_species.particle_type)
```

Comment:
It would be nice here to check if an environment variable is specified to give the warpx-data directory. It won't always be at this relative path.

Reply:
For ease, I think this is the code:
```suggestion
        path_name = os.environ.get("MCC_CROSS_SECTIONS_DIR", os.path.join("../../../warpx-data/MCC_cross_sections", self.ion_species.particle_type))
```
With line breaks...

Reply:
Also instead of `os.path.join("../../../warpx-data/MCC_cross_sections", self.ion_species.particle_type)` use `os.path.join(mwxutil.mewarpx_dir, "../../warpx-data/MCC_cross_sections", self.ion_species.particle_type)`